spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/mapper/ChebiToUniProtDocument.java

```diff
@@ -80,4 +50,64 @@ public UniProtDocument call(Tuple2<UniProtDocument, Optional<Iterable<ChebiEntry
         }
         return doc;
     }
+
```

Comment:
Nice that you've refactored the code to put them in two new methods. Ground is more cleaner now :-)